### DISCUSSION TITLE
Idle replicas vs min replicas

### DISCUSSION THREAD

**Discussion by RomanDvorsky** [kedacore/keda#2610] [General] [answered]
Hi, 

what is the reason that idle replicas have to be lower than min replicas? It can be only 0 in the current setup other values do not make sense to me.

I have service with min replicas: 2 when I want to use KEDA to scale by the number of requests and do not want to shut down my service I have to set idle replicas to 1 otherwise KEDA will kill my service during the night (or throw an error in case min replicas = idle replicas).

However, this setup will start flapping instances. HPA behind KEDA will scale to 2 instances and KEDA will scale to 1 instance and kill HPA setup, HPA will scale up again....

Thx for the explanation,
R.

**Comment by zroubalik** [comment, marked as answer]
You don't have to define idle replicas. It's optional. If you don't set, it will be ignored.

You can set, min to 1 and max to whaterver number and replicas will scaled between this numbers.

**Reply by RomanDvorsky** [reply]
thx for the explanation

**Reply by tomkerkhove** [reply]
Is there anything that we can improve in our docs on this @RomanDvorsky ?

**Reply by RomanDvorsky** [reply]
Hi @tomkerkhove,

maybe just a little not that it is ignored when not specified - I get an impression that it will take 0 as a default value.

also one little note - would be great if it is possible to make a link to each property like: https://keda.sh/docs/2.6/concepts/scaling-deployments/#idleReplicaCount and not only to whole section like it is now: https://keda.sh/docs/2.6/concepts/scaling-deployments/#details 

Regards,
R.

**Reply by tomkerkhove** [reply]
This is actually documented on https://keda.sh/docs/latest/concepts/scaling-deployments/#details, but it looks like you've used a KEDA v1.x URL.

Can you verify that the current docs on this are better?

**Reply by tomkerkhove** [reply]
> also one little note - would be great if it is possible to make a link to each property like: [keda.sh/docs/2.6/concepts/scaling-deployments/#idleReplicaCount](https://keda.sh/docs/2.6/concepts/scaling-deployments/#idleReplicaCount) and not only to whole section like it is now: [keda.sh/docs/2.6/concepts/scaling-deployments/#details](https://keda.sh/docs/2.6/concepts/scaling-deployments/#details)

Sure thing! We are open for PRs, if you want to help us with that?

**Reply by RomanDvorsky** [reply]
here you are: https://github.com/kedacore/keda-docs/pull/651
